README.md

```diff
@@ -5,7 +5,7 @@ This example contains a ready data access layer generated with Entity Framework
 
 Database structure:
 
-![](/Images/DatabaseStructure.png)
```

Comment:
"This example contains a ready data access layer" What is a **ready data access layer**?

Reply:
"Entity Framework Database First" First?

Reply:
"to the freely distributed" Why do you add 'freely'?

Reply:
or add "This database is commonly used for demonstration purposes"? What is the purpose of this sentence?